web/layout.html

```diff
@@ -33,8 +33,9 @@
         headers.set('Authorization', 'Basic ' + window.btoa(unescape(encodeURIComponent(authString))))
       }
 
-      const response = await window.fetch("/tdexdconnect", { method: 'GET', headers,});
+      const response = await window.fetch(`${window.location.pathname}tdexdconnect`, { method: 'GET', headers,});
```

Comment:
Not sure this is always correct. I tried out `window.location.pathname` and when it's not `/`, it retuns a string like `/path/example`. I think the best thing to do is making sure it's value ends with `/`:
```suggestion
      const pathname = window.location.pathname.endsWith('/') ? window.location.pathname : `${window.location.pathname}/`
      const response = await window.fetch(`${window.location.pathname}tdexdconnect`, { method: 'GET', headers,});
```